src/lib/aws/timestream_manager.py

```diff
@@ -0,0 +1,85 @@
+import boto3
+
+from datetime import datetime
+import time
+
+
+def iso_time_to_epoch_milliseconds(iso_date: str) -> str:
+    """
+    Convert an ISO 8601 formatted date string to the number of milliseconds since the Unix epoch.
+    If the input is None, the current time in milliseconds since the Unix epoch is returned.
+
+    Parameters:
+    iso_date (str): An ISO 8601 formatted date string (e.g., "2023-11-21T21:39:09Z").
+                    If None, the current time is used.
+
+    Returns:
+    str: The number of milliseconds since the Unix epoch as a string.
+    """
+
+    # If the input is None, use the current time
+    if iso_date is None:
+        epoch_time = time.time()
+    else:
+        # Convert the ISO date string to a datetime object
+        dt = datetime.fromisoformat(iso_date.rstrip("Z"))
+        # Convert the datetime object to epoch time in seconds
+        epoch_time = dt.timestamp()
+
+    # Convert epoch time to milliseconds and return as string
+    return str(int(epoch_time * 1000))
+
+
+class TimestreamTableWriter:
```

Comment:
What do you think about renaming this class to TimestreamManager? And in future we will be able to add new methods connected to Timestream if needed.

Reply:
Actually, that was the idea to name like that.
In future we will have, I guess, TimestreamTableReader, which is significantly different.
It even uses another boto3 client (there are separate clients for timestream write and read).